populate_songs.py

```diff
@@ -27,9 +28,11 @@ def get_full_song_name(spreadsheetId, inSongName: str):
         if normalize_song_name(songName) == searchKey:
             return songName
 
-def populate_songs(spreadsheetId, inEvents, shouldPrintTitle):
+def populate_songs(spreadsheetId, inEvents, shouldPrintTitle, shouldDelay):
     outEvents = []
 
+    hasStarted = False
```

Comment:
Not quite sure what this variable is for...

Reply:
It was used to not delay on the very first song, but now it's no longer necessary and has been removed.